lbz/request.py

```diff
@@ -18,7 +18,7 @@
 class Request:
     """Represents request from API gateway."""
 
-    _json_body = None
+    _json_body: dict = {}
```

Comment:
You shouldn't assign an empty dict here. That is the class attribute, so it will remain as long as the code is running. Consider this example:
```python
>>> class Request:
...     _json_body = {}
...     
...     def json_body(self):
...         return self._json_body
... 
...     def do_smth(self):
...         self._json_body["test"] = "test"
>>> a = Request()
>>> a.json_body()
{}
>>> a.do_smth()
>>> a.json_body()
{'test': 'test'}
>>> b = Request()
>>> b.json_body()
{'test': 'test'}
```

Reply:
In general, together with `_raw_body`, it should be declared inside `__init__`.